deploy/full-lifecycle.Jenkinsfile

```diff
@@ -7,7 +7,7 @@ node {
             result = build job: params.BuildLegionJobName, propagate: true, wait: true, parameters: [
                     [$class: 'GitParameterValue', name: 'GitBranch', value: params.GitBranch],
                     booleanParam(name: 'PushDockerImages', value: true),
-                    booleanParam(name: 'EnableDockerCache', value: false),
+                    booleanParam(name: 'EnableDockerCache', value: true),
```

Comment:
You can not use this parameter now, it is obsollete

Reply:
I see the parameter implemented across all our pipelines and jobs, so guess it worth to refactor them in a separate story, just made a proper use of that for the purpose of the build process here